config/tests/testgrids/config_test.go

```diff
@@ -442,7 +442,7 @@ func TestKubernetesProwInstanceJobsMustHaveMatchingTestgridEntries(t *testing.T)
 	for testgroup, valid := range testgroups {
 		if !valid {
 			badconfigs = append(badconfigs, testgroup)
-			t.Errorf("Testgrid group %v does not have a matching jenkins or prow job", testgroup)
+			t.Errorf("Testgrid group %v is supposed to be moved to have their presubmits in testgrid. See TODO here: https://github.com/kubernetes/test-infra/blob/master/config/tests/testgrids/config_test.go#L346-L348", testgroup)
```

Comment:
Instead of linking to lines of code which are likely to drift over time, can you open and link to an issue for this task?

Reply:
Sure will do.